NEWS.md

```diff
@@ -40,14 +40,20 @@
 - Sets the `UVICORN_ROOT_PATH` environment variable to the proxied port URL for port 8000 in VS Code and Jupyter sessions, allowing FastAPI applications to run without additional configuration. (rstudio-pro#2660)
 - Handles `SIGTERM` signals to shut down more gracefully, for better interaction with service managers like `systemd`.
 
+#### Posit Workbench VS Code Sessions
+
+- Install VS Code session support (code-server) with Posit Workbench instead of requiring a separate download (rstudio-pro#3643)
+- Enable VS Code sessions by default on initial Workbench install (rstudio-pro#3643)
+- Updated code-server to version 4.7.1 (VS Code version 1.71.2) (rstudio-pro#3643)
```

Comment:
```suggestion
- Updated code-server to version 4.7.1 (VS Code version 1.71.2) (rstudio-pro#3643)
- Sets the `UVICORN_ROOT_PATH` environment variable to the proxied port URL for port 8000 in VS Code and Jupyter sessions, allowing FastAPI applications to run without additional configuration. (rstudio-pro#2660)
```
Since we're adding a VS Code section, would you mind moving this note from above here as well?

Reply:
Will do!